ntoskrnl/mm/ARM3/expool.c

```diff
@@ -487,6 +487,8 @@ ExpTagAllowPrint(CHAR Tag)
     DPRINT1(fmt, ##__VA_ARGS__)
 #endif
 
+ULONGLONG MiLastPoolDumpTime;
```

Comment:
May be set as a `static ULONGLONG MiLastPoolDumpTime = 0;` inside `ExAllocatePoolWithTag()` in case that variable is not used elsewhere?

Reply:
My thought was (1) our kernel coding style doesn't usually do this, (2) that wouldn't allow dumping the value in the debugger.

Reply:
And (3), sometimes gcc does do funny things with those statics.

Reply:
Ok, point (2) is good; I'm wondering what kind of funny things GCC does (point 3)?